sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_helpers.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from datetime import time
+from azure.core.exceptions import HttpResponseError
+
+POLLING_INTERVAL = 1
+
+
+def get_pipeline_response(pipeline_response, _, response_headers):  # pylint: disable=unused-argument
+    return pipeline_response
+
+
+def get_field_scalar_value(field):  # pylint: disable=too-many-return-statements
+    field_type = field.type
+    if field_type == "string":
+        return field.value_string
+    if field_type == "number":
+        return field.value_number
+    if field_type == "integer":
+        return field.value_integer
+    if field_type == "date":
+        return field.value_date
+    if field_type == "phoneNumber":
+        return field.value_phone_number
+    if field_type == "time":
+        try:
+            hour, minutes, seconds = field.value_time.split(":")
+            return time(int(hour), int(minutes), int(seconds))
+        except AttributeError:  # service recognized field as time but returned None for value because it wasn't valid
+            pass
+    return None
```

Comment:
Is this ever expected? If not, it should be an exception...

Reply:
Need to think about this more. Not sure I want to raise an exception here.